src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -5512,15 +5503,8 @@ namespace CondenserLoopTowers {
         }
 
         WaterMassFlowRatePerCell = this->WaterMassFlowRate / this->NumCellOn;
-        // MassFlowTolerance is a parameter to indicate a no flow condition
-        if (this->WaterMassFlowRate <= DataBranchAirLoopPlant::MassFlowTolerance || (MyLoad > HVAC::SmallLoad)) {
-            // for multiple cells, we assume that it's a common bassin
-            CalcBasinHeaterPower(
-                state, this->BasinHeaterPowerFTempDiff, this->BasinHeaterSchedulePtr, this->BasinHeaterSetPointTemp, this->BasinHeaterPower);
-            return;
-        }
 
-        if (std::abs(MyLoad) <= HVAC::SmallLoad) {
+        if ((std::abs(MyLoad) <= HVAC::SmallLoad) || !RunFlag) {
```

Comment:
I am wondering if just !RunFlag would meet the intent. I would think RunFlag would be false if (std::abs(MyLoad) <= HVAC::SmallLoad) but then I am not sure.

Reply:
I wasn't sure either and given that the Merkel model already used MyLoad, I felt like having both would be "safer".  My guess is that both of these flags will probably trip this IF at the same time, but I didn't see anything wrong with doing it this way which seemed like it would catch all conditions.